src/rgw/rgw_rados.cc

```diff
@@ -12398,8 +12398,7 @@ int RGWRados::list_bi_log_entries(RGWBucketInfo& bucket_info, int shard_id, stri
   librados::IoCtx index_ctx;
   map<int, string> oids;
   map<int, cls_rgw_bi_log_list_ret> bi_log_lists;
-  map<int, string> bucket_instance_ids;
-  int r = open_bucket_index(bucket_info, index_ctx, oids, shard_id, &bucket_instance_ids);
+  int r = open_bucket_index(bucket_info, index_ctx, oids, shard_id);
```

Comment:
I think that is intentional to make [this](https://github.com/ceph/ceph/blob/28c79fac6eb580dbb3a1bbb05c77a95ab75c277c/src/rgw/rgw_rados.cc#L8566) work.

Reply:
@joscollin can you please take another look? the line you linked allows the caller to request the bucket instance ids if they need them. since we don't use them here, we can skip the argument and avoid the work of allocating a map and formatting strings for the ids

Reply:
@cbodley Yes, the caller doesn't use it.